CHANGELOG.md

```diff
@@ -4,9 +4,14 @@
 
 ## Important Notes
 
+- [#1086](https://github.com/oauth2-proxy/oauth2-proxy/pull/1086) The extra validation to protect invalid session deserialization
```

Comment:
Rationale:

SHA256 was added in v6.0.0
LZ4 + MessagePack was added in v6.1.0

Underlying encryption is still AES-CFB which is unauthenticated where the SHA is our MAC. A session created in v6.0.0 could be SHA256 MAC, then CFB decrypt, and old style JSON would possible pass LZ4 decompression & messagepack might see leading bytes and unmarshal an empty `Session` instead of erroring.